src/main/java/com/github/fppt/jedismock/operations/scripting/cjson/LuaCjsonLib.java

```diff
@@ -0,0 +1,18 @@
+package com.github.fppt.jedismock.operations.scripting.cjson;
+
+import org.luaj.vm2.LuaTable;
+import org.luaj.vm2.LuaValue;
+import org.luaj.vm2.lib.TwoArgFunction;
+
+public class LuaCjsonLib extends TwoArgFunction {
+
+    @Override
+    public LuaValue call(LuaValue modname, LuaValue env) {
+        LuaTable cjson = new LuaTable();
```

Comment:
If I get this right, we could override the `set` method of `LuaTable` in order to lock it from modifications, thus making `cjson` an immutable table. `encode` and `decode` values could be set in constructor via `super.set(...)`. This would make `evalCjsonIsReadOnlyTable` test green. But maybe you deleberately want users to substitute functions in `cjson` for testing purposes?

Reply:
I'm not sure if it's really needed, since I don't see any use cases for this
> But maybe you deleberately want users to substitute functions in cjson for testing purposes?

Sure! I'll apply the changes
> 
If I get this right, we could override the set method of LuaTable in order to lock it from modifications, thus making cjson an immutable table.